crates/flox-rust-sdk/src/models/publish.rs

```diff
@@ -160,24 +160,22 @@ impl<'flox> Publish<'flox, Empty> {
         ]
         .to_vec();
 
-        if self.stability != Stability::Unspecified {
-            let nixpkgs_flakeref = FlakeRef::Indirect(IndirectRef::new(
-                format!("nixpkgs-{}", self.stability),
-                Default::default(),
-            ));
-
-            // Stabilities are managed by overriding the `flox-floxpkgs/nixpkgs/nixpkgs` input to
-            // `nixpkgs-<stability>`.
-            // The analyzer flake adds an additional indirection,
-            // so we have to do the override manually.
-            // However, since https://github.com/flox/flox/pull/182,
-            // we only set this when a stability is specified
-            // This is the `nixpkgs-<stability>` portion.
-            override_inputs.push(OverrideInput {
-                from: "target/flox-floxpkgs/nixpkgs/nixpkgs".to_string(),
-                to: nixpkgs_flakeref,
-            });
-        }
+        let nixpkgs_flakeref = FlakeRef::Indirect(IndirectRef::new(
```

Comment:
It's a bit misleading to make changes to stability in a PR that's mostly supposed to be a refactor. Are those needed in this PR? Can you add some more explanation in some of the commits like this one https://github.com/flox/flox/pull/215/commits/556e013b7da2527e4aa4ff91d6e197e8d8d7bff9?

Reply:
Agreed, I thought that there weren't any functional changes?

Reply:
The stability stuff had to be refactored somewhat as before, we were depending on the grouping to override them for all `Package`/"Development" commands.
They are now applied per command.

The publish changes were asked for by @limeytexan.

I can squash the stability commits if that makes it easier.
I would have preferred PRing them separately, just didn't want to break stabilities in the meantime.
We can also try keep the original behavior of publish and change that in a separate PR (maybe that's a better way to order that...)